experiments/correlation/sweep_upsample_factor.py

```
@@ -0,0 +1,66 @@
+import numpy as np
+from typing import Tuple, List
```

Ruff (0.14.3)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)